docs/threat_model/threat_model.rst

```diff
@@ -337,14 +337,14 @@ What a compromise of the journalist's property can achieve
    journalist's Tails device can:
 
    -  Add, modify, and delete files on the volume.
-   -  Access the Hidden Service values used by the Journalist Interface.
+   -  Access the Hidden Service values used by the *Journalist Interface*.
    -  Access SSH keys and passphrases for the *Application Server* and the
       *Monitor Server*.
    -  Access the journalist's personal GPG key.
 
 -  An attacker with journalist access to the *Journalist Interface* can:
 
-   -  Change the codenames associated with sources within the Interface.
+   -  Change the codenames associated with sources within the interface.
```

Comment:
I don't think we should use a capitalize form here.

Reply:
I agree. The diff looks like I did make it lowercase, but I'll check again.

Reply:
Oh, you are right! I read it the other way around, my bad.